test/cri-containerd/logging_binary_test.go

```diff
@@ -0,0 +1,138 @@
+// +build functional
+
+package cri_containerd
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+// This test requires compiling a helper logging binary which can be found
+// at test/cri-containerd/helpers/log.go. Copy log.exe as "sample-logging-driver.exe"
+// to ContainerPlat install directory or set "TEST_BINARY_ROOT" environment variable,
+// which this test will use to construct logPath for CreateContainerRequest and as
+// the location of stdout artifacts created by the binary
+func Test_Run_Container_With_Binary_Logger(t *testing.T) {
```

Comment:
Can we have some mechanism to bail out on this test if the machine doesn't have the necessary bits? E.g. can we stat for the logging binary and if it's not present skip the test?

Reply:
yup makes sense